app/bundles/LeadBundle/Views/Auditlog/details.html.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+$details = $event['details'];
+$type    = $event['eventType'];
+$text    = '';
+switch ($type) {
+    case 'create':
+        $text = 'The contact was created.';
+        if (isset($details['ipAddresses']) && is_array($details['ipAddresses'])) {
+            $text .= ' Origin IP: '.implode(', ', array_splice($details['ipAddresses'], 1));
+        }
+        break;
+    case 'delete':
+        $text = 'The contact was deleted.';
```

Comment:
All these hard coded strings in English need to be converted to translation strings.